lib/game_box_web/live/arena_live.ex

```diff
@@ -362,4 +355,14 @@ defmodule GameBoxWeb.ArenaLive do
       player_id: player_id
     })
   end
+
+  defp populate_subtext(game_selected, is_host) do
```

Comment:
I think we could split these out into function definitions that pattern matches the game_selected argument being `nil` and the is_host value. Maybe something like:

```
defp populate_subtext(nil, true), do: "Select.."
defp populate_subtext(nil, false), do: "Waiting for.."
defp populate_subtext(_, _), do: nil
```

Reply:
Oh yeah that's nicer. Done.